aws/logs_monitoring/template.yaml

```diff
@@ -229,6 +229,10 @@ Parameters:
     Type: String
     Default: ""
     Description: The name of the forwarder bucket to create. If not provided, AWS will generate a unique name.
+  DdForwarderExsitingBucketName:
+    Type: String
+    Default: ""
+    Description: The name of an existing s3 bucket to use. If not provided, a new bucket will be created.
```

Comment:
Do we decide on a specific naming when creating it ? We may want to explicit that here (in case there are conflicts)